### ISSUE TITLE
archarm borked 

### ISSUE THREAD

**Issue by Rippanda12** [westandskif/rate-mirrors#33]
Server = http://gr.mirror.archlinuxarm.org/$arch/$arch/$repo
Server = http://fl.us.mirror.archlinuxarm.org/$arch/$arch/$repo
 it gives $arch twice should be just once

**Comment by Rippanda12** [comment]
```# STARTED AT: 2022-08-27 22:28:31.660199787 +03:00
# ARGS: rate-mirrors archarm
# MIRRORS LEFT AFTER FILTERING: 21
# JUMP #1
# EXPLORING US
# VISITED US
# BLANK ITERATION
# COUNTRY JUMPING YIELDED TOO FEW MIRRORS (0), ADDING OTHERS TO UNLABELED
# 

# TESTING UNLABELED MIRRORS
# SpeedTestResult { speed: 723.98 KB/s; elapsed: 375.6217ms; connection_time: 381.413949ms }
# SpeedTestResult { speed: 318.98 KB/s; elapsed: 269.900931ms; connection_time: 1.218138959s }
# SpeedTestResult { speed: 924.40 KB/s; elapsed: 1.148671216s; connection_time: 343.938229ms }
# SpeedTestResult { speed: 316.39 KB/s; elapsed: 971.775526ms; connection_time: 528.532782ms }
# SpeedTestResult { speed: 410.48 KB/s; elapsed: 1.015014958s; connection_time: 470.083921ms }
# SpeedTestResult { speed: 269.86 KB/s; elapsed: 679.942099ms; connection_time: 820.426376ms }
# SpeedTestResult { speed: 224.11 KB/s; elapsed: 714.289841ms; connection_time: 752.487035ms }
# 

# RE-TESTING TOP MIRRORS
# SpeedTestResult { speed: 786.36 KB/s; elapsed: 500.627501ms; connection_time: 583.801602ms }
# SpeedTestResult { speed: 2.16 MB/s; elapsed: 375.599556ms; connection_time: 334.315909ms }
# SpeedTestResult { speed: 2.51 MB/s; elapsed: 607.240391ms; connection_time: 747.290204ms }
# SpeedTestResult { speed: 1.08 MB/s; elapsed: 355.652773ms; connection_time: 1.144203551s }
# SpeedTestResult { speed: 699.12 KB/s; elapsed: 591.104246ms; connection_time: 374.952819ms }
# ==== RESULTS (top re-tested) ====
#   1. SpeedTestResult { speed: 2.51 MB/s; elapsed: 607.240391ms; connection_time: 747.290204ms } -> http://de3.mirror.archlinuxarm.org/
#   2. SpeedTestResult { speed: 2.16 MB/s; elapsed: 375.599556ms; connection_time: 334.315909ms } -> http://hu.mirror.archlinuxarm.org/
#   3. SpeedTestResult { speed: 1.08 MB/s; elapsed: 355.652773ms; connection_time: 1.144203551s } -> http://mirror.archlinuxarm.org/
#   4. SpeedTestResult { speed: 786.36 KB/s; elapsed: 500.627501ms; connection_time: 583.801602ms } -> http://de4.mirror.archlinuxarm.org/
#   5. SpeedTestResult { speed: 699.12 KB/s; elapsed: 591.104246ms; connection_time: 374.952819ms } -> http://gr.mirror.archlinuxarm.org/
#   6. SpeedTestResult { speed: 269.86 KB/s; elapsed: 679.942099ms; connection_time: 820.426376ms } -> http://fl.us.mirror.archlinuxarm.org/
#   7. SpeedTestResult { speed: 224.11 KB/s; elapsed: 714.289841ms; connection_time: 752.487035ms } -> http://nj.us.mirror.archlinuxarm.org/
# FINISHED AT: 2022-08-27 22:28:39.822902884 +03:00
Server = http://de3.mirror.archlinuxarm.org/$arch/$arch/$repo
Server = http://hu.mirror.archlinuxarm.org/$arch/$arch/$repo
Server = http://mirror.archlinuxarm.org/$arch/$arch/$repo
Server = http://de4.mirror.archlinuxarm.org/$arch/$arch/$repo
Server = http://gr.mirror.archlinuxarm.org/$arch/$arch/$repo
Server = http://fl.us.mirror.archlinuxarm.org/$arch/$arch/$repo
Server = http://nj.us.mirror.archlinuxarm.org/$arch/$arch/$repo
```
full log

**Comment by westandskif** [comment]
fixed in v0.11.1
